### ISSUE TITLE
[BUG] No way to set aria-label for kirby-menu default button

### ISSUE THREAD

**Issue by Woobs8** [kirbydesign/designsystem#3916]
## Describe the bug
The default `<kirby-menu>` button does not a have an aria-label by default, and there is seemingly no way to set it explicitly. The only workaround appears to be creating a custom menu button.

## Describe how to reproduce the bug
Any `kirby-menu` with a default button will reproduce the error.

## Which Kirby version was used?
10.3.1

## What was the expected behavior?
The default menu button should either have a default aria-label and/or provide a way to set it explicitly.

## Please complete the following information:
- OS: Windows, MacOS, iOS, Android
- Browser [e.g. Chrome, Safari]: Chrome, Safari, Firefox

## Are there any additional context?
This issue directly affects the ability to properly support screen readers.

<hr />

## Checklist:

The following tasks should be carried out in sequence in order to follow [the process of contributing](https://github.com/kirbydesign/designsystem/blob/develop/.github/CONTRIBUTING.md#the-process-of-contributing) correctly.

### Verification
To make sure the bug is not intended behaviour; it should be verified by a member of team Kirby before moving on to implementation. 

- [ ] Make sure the `NOT verified` label has been removed by a member of [team Kirby](https://github.com/kirbydesign/designsystem/blob/develop/.github/SUPPORT.md#team-kirby); do not proceed until this is done.

### Implementation 
The contributor who wants to implement this issue should: 

- [ ] Make sure you have read: "[Before you get coding](https://github.com/kirbydesign/designsystem/blob/develop/.github/CONTRIBUTING.md#before-you-get-coding)".
- [ ] Signal to others you are working on the issue by assigning yourself.
- [ ] Create a branch from the [develop branch](https://github.com/kirbydesign/designsystem/tree/develop) following our [branch naming convention](https://github.com/kirbydesign/designsystem/wiki/The-Good%3A-Branch). 
- [ ] Create a test that reproduces the bug following guidelines in: "[The good: Test](https://github.com/kirbydesign/designsystem/wiki/The-Good%3A-Test)". 
- [ ] Write code to fix the bug

### Review
Once the issue has been implemented and is ready for review:

- [ ] Do a [self-review](https://github.com/kirbydesign/designsystem/wiki/The-Good%3A-Self-review). 
- [ ] Create a pull-request.

**Comment by dennispedersen-bd** [comment]
Screen reader should read it aloud as "**Mere. Knap.**" in Danish. Meaning that the default aria label for menu buttons with 3 dots should be:

DA  = "Mere"
EN = "More"
DE = "Mehr"
